examples/fdinteractive_fitter.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[42]:
+
+
```

Comment:
you can just delete these blank lines (at the very least, the commented-out line 4)